examples/Makefile.am

```diff
@@ -32,7 +27,9 @@ pkginclude_HEADERS =
 #
 # CPP TESTS
 #
+antioch_initdir = $(prefix)/share/examples/antioch_init
 antioch_init_SOURCES = antioch_init.C
+antioch_init_DATA    = ${antioch_init_SOURCES}
 
 #Define examples to actually be run
 TESTS  =
```

Comment:
Nuke this and the associated lines below it.  This doesn't belong to the test suite.